census_extractomatic/api.py

```diff
@@ -35,14 +36,18 @@
 db = SQLAlchemy(app)
 sentry = Sentry(app)
 
-app.s3 = S3Connection()
-
 if not app.debug:
     import logging
     file_handler = logging.FileHandler('/tmp/api.censusreporter.org.wsgi_error.log')
     file_handler.setLevel(logging.WARNING)
     app.logger.addHandler(file_handler)
 
+try:
+    app.s3 = S3Connection()
+except Exception, e:
+    app.s3 = None
+    app.logger.warning("S3 Configuration failed.")
```

Comment:
Can this be `app.logger.exception()` so that we get the exception logged out in addition to the message? That would help with debugging if something is broken.